ui/client/components/graph/node-modal/EdgeDetailsContent.js

```diff
@@ -57,37 +58,34 @@ export default class EdgeDetailsContent extends React.Component {
   }
 
   render() {
-    const { edge, readOnly, updateEdgeProp, showValidation} = this.props
+    const {edge, readOnly, updateEdgeProp, showValidation, showSwitch} = this.props
 
     switch (_.get(edge.edgeType, 'type')) {
       case "SwitchDefault": {
         return this.baseModalContent()
```

Comment:
Please do simple functional component: `const BaseModal Content = (props) => {}`. In props you should alsow have a children element for rendering it at NextSwitch.

Reply:
Done